src/Microsoft.Data.SqlClient/tests/Common/Common.csproj

```diff
@@ -51,4 +51,10 @@
     </ContentWithTargetPath>
   </ItemGroup>
 
+  <!-- Fixture-specific References -->
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="$(AzureIdentityVersion)" />
```

Comment:
Use central package management here.

Reply:
Thanks @cheenamalhotra. I've just fixed this part of the review - I'm not sure what the best approach would be around the `TargetNetCoreVersion` point, so have left it alone.